js/popover.js

```diff
@@ -64,9 +64,22 @@
   Popover.prototype.getContent = function () {
     var $e = this.$element
     var o  = this.options
+    var contentType = typeof o.content
+
+    if (!/string|function/.test(contentType)) {
+      throw new Error('Popover received type ${' + contentType + '}. Was expecting type string or a function.')
```

Comment:
`${}` are part of the ES6 interpolation syntax. I was just trying to make clear where the variable should be included :smile: